### DISCUSSION TITLE
Multiple Threads and More Memmory per plot

### DISCUSSION THREAD

**Discussion by ebeng** [Chia-Network/chia-blockchain#4561] [Support]
Hi all,

Trying to tweak out what I possible could do to get more out of the system. 
Running on a DELL server, I've managed to get CHIA work.

Setup as follows; 
VMWARE 6.7 ; UbuntuDesktop-20.04 ; CPU 32x, 64GB Mem and 7TB of disk attached (datastore where it uses 2x 3.8TB SATA SSD). 

I do see that when running, 14 parallel plots, it consumes all the (80-95%) CPU of the VM and the disk writes/reads are in total around the 800-1000 MBps.

However, I did start now 5 plots, where 2 of them differs;
- 3x Standard deployment
- 1x 6780Mem and 4Threads
- 1x 13560Mem and 8 Threads

The total amount of CPU is now on 20-25% and the disk read/write in total around the 450MBps.
I can clearly see that the one with 8Threads is going to finish earlier than the others as it is showing higher percentage.

What does this mean? Just run 14 parallel with standard definitions? Or split the VM into 2 where it does the job on 2 plotters where it can access 2 SSD directly?

**Comment by ebeng** [comment]
Anyone :)?

**Comment by BrandtH22** [comment]
@ebeng , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - over the past couple years more advanced plotters have been released with enhanced features. More information about these can be found here https://docs.chia.net/plotting-software

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
